test/tools/isoltest.cpp

```diff
@@ -189,12 +189,24 @@ TestTool::Result TestTool::process()
 	}
 	catch (boost::exception const& _e)
 	{
+		auto const* semanticTest = dynamic_cast<SemanticTest const*>(m_test.get());
+		if (semanticTest && semanticTest->runningEnforcedViaYul())
+		{
+			AnsiColorized(cout, formatted, {BOLD, GREEN}) << "OK" << endl;
+			return Result::Success;
+		}
```

Comment:
I didn't find better solution for this. If there is missing feature in sol2yul it usually asserts which ends up here.
When doing forced run of compilation via yul it sometimes triggers exception...

@axic if you have any better idea, it is very welcome

Reply:
Also, enforced run is last one in the chain for particular test, so it is fine to return success in this case